workers/data_refinery_workers/processors/test_array_express.py

```diff
@@ -201,6 +203,83 @@ def test_success(self):
         # Clean up the copied file
         os.remove(input_file_path)
 
+    def test_pcl_is_valid(self):
+        batch = init_objects()
+        file = batch.files[0]
```

Comment:
Hah, I was going to ask if we should be using the `file` builtin in Python as a variable name, but I guess it's not even a builtin in Python3. Do we still want to give it a more descriptive name, though, like `scan_output_file`, `output_pcl_file` or something like that? I feel like `file` is too ambiguous.

Reply:
I kinda think `file`'s ambiguity is useful. I am being to consider that perhaps a more powerful model may be necessary, but in the current data model the File class has metadata about actual files associated with Batches. `scan_output_file` and `output_pcl_file` are not accurate names for this, because prior to processing `file` refers to unprocessed data and then after processing refers to processed data. The only accurate name I can think of that might be more descriptive is something like `the_file_of_the_batch` which is somewhat obvious from the context.

If you still think it's ambiguity affects the code clarity I can change it, but I've been using `file`, `file1`, and `file2` all over the place as a psuedo-convention whereby it refers to instances of `File` objects when there are no other `File` objects it could be confused with.

Reply:
Up to you. I feel like it would help someone else reading the code, if it was something like `batch_pcl_file`, since my Pythonic sense is tingling looking at just `file`. But if you feel strongly about the reason for it, it's fine.

Reply:
`batch_pcl_file` isn't accurate, because it's not a PCL until after it's been processed, until then it's a CEL. I could maybe name the variable name `batch_file`. Do you think that's more clear?

Reply:
Mmm. Ok, that's fine, if you feel like the ambiguity is working in the favor of this code then I think it's fine to keep it like that.

Reply:
I've added a comment above the variable declaration to clarify as much as possible.